src/core/Tooltip/Tooltip.tsx

```diff
@@ -68,6 +59,6 @@ export const Tooltip = (props: TooltipProps) => {
       {children && React.cloneElement(children, { title: undefined })}
     </Popover>
```

Comment:
Does adding the ref in reference={ref} instead of ref in popover make it work ?

Reply:
That's not how `reference` is supposed to be used